scripts/install_bootstrap

```diff
@@ -1,6 +1,10 @@
 #! /bin/bash
 set -e
 PANCANCER_LAUNCHER_VERSION=3.1.7
+PANCANCER_CLI_VERSION=0.0.7
+WORKFLOW_LISTING_URL=https://github.com/ICGC-TCGA-PanCancer/cli/blob/${PANCANCER_CLI_VERSION}/config/workflowlist.json
```

Comment:
Ya, I think longer term the workflow JSON will be hosted on pancancer.info perhaps?